start-tcp.sh

```diff
@@ -3,7 +3,4 @@ if [ $1 = "bbr" ] ; then
 	sudo tc qdisc add dev ingress root fq pacing
 	sudo tc qdisc show dev ingress
 fi
-iperf -c $MAHIMAHI_BASE -p 42425 -w 16m -t 140 -Z $1 &
-sleep 140
-killall iperf
-killall iperf
+iperf -c $MAHIMAHI_BASE -p 42425 -w 16m -t 1000 -Z $1
```

Comment:
Why do we no longer need to run processes in the background?

Reply:
This command is run inside a mahimahi shell, and we wait for that process to finish.  The shell will exit once the trace has been played once, so this should stay in the foreground